src/main/java/org/gradle/profiler/studio/launcher/LauncherConfigurationParser.java

```diff
@@ -67,6 +108,8 @@ public LaunchConfiguration calculate() {
         commandLine.add("-cp");
         commandLine.add(Joiner.on(File.pathSeparator).join(classpath));
         systemProperties.forEach((key, value) -> commandLine.add(String.format("-D%s=%s", key, value)));
+        // Support for IntelliJ with Java17 runtime
+        commandLine.addAll(JAVA17_ADD_OPENS);
```

Comment:
Should we always add these, or only when Java version is >= 17?

Reply:
I tested with different Android Studio versions that have Java11 runtime and it works. I believe these modules are anyway accessible in Java11 by default.

We could open it just for Java17, but it means we need to figure out what JDK version an IDE has bundled (probably for every operating system separately) before we run it. So I would add it always unless we find out there are problems with it.